coala_quickstart/Strings.py

```diff
@@ -50,3 +50,9 @@
 
 You can see all of them here: {}
 """.format(BEAR_DOCS_URL)
+
+GENERATOR_INFO = """
+# coala-quickstart is a tool that helps users to quickly get started with coala.
+# It can generate a .coafile (coala's configuration file) that is tailored
+# perfectly for your project without any interaction.
```

Comment:
I do not see why we need this string.  Typical practise is to put one line, at the top, to remind the user how and when the file was created.

A generated .coafile will never be ideal -- in any serious project it will be edited by a human before being checked into VCS, and this header will likely be removed.